common-app-lib/src/cmr/common_app/services/search/elastic_search_index.clj

```diff
@@ -19,18 +19,30 @@
    [cmr.elastic-utils.config :as es-config]
    [cmr.elastic-utils.connect :as es]
    [cmr.elastic-utils.es-helper :as es-helper]
-   [cmr.transmit.connection :as transmit-conn])
+   [cmr.transmit.connection :as transmit-conn]
+   [cmr.common.config :as cfg :refer [defconfig]])
   (:import
    clojure.lang.ExceptionInfo
    java.net.UnknownHostException))
 
+(defconfig collections-index-alias
+           "The alias to use for the collections index."
+           {:default "collection_search_alias" :type String})
```

Comment:
string is the default, I think we normally drop the type in this case.

Reply:
This was pre-existing code I moved into this file. I think I would like to keep it as is for this.